RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -561,35 +561,7 @@ boolean handleBjornify(familiar fam)
 
 void equipBaseline()
 {
-	if(my_daycount() == 1)
-	{
-		if(have_familiar($familiar[grimstone golem]))
-		{
-			if(get_property("_grimFairyTaleDropsCrown").to_int() >= 1)
-			{
-				handleBjornify($familiar[El Vibrato Megadrone]);
-			}
-		}
-		else
-		{
-			handleBjornify($familiar[El Vibrato Megadrone]);
-		}
-	}
-	if(my_daycount() == 2)
-	{
-		handleBjornify($familiar[El Vibrato Megadrone]);
-	}
-
-	if(get_property("auto_diceMode").to_boolean())
```

Comment:
Should we remove `auto_diceMode` from settings?

Reply:
I'll look into auto_diceMode, never used it myself as my test account hasn't done any OCRS runs yet but I think OCRS needs some testing too as there's some stuff for the Clingy modifier which I think is also unnecessary due to mafia fixes for chained combats.